packages/api-specification/src/window-api-demo.js

```diff
@@ -2,14 +2,45 @@ var newWindowButton = document.getElementById('new-window-test');
 
 const appReady = ssf.app.ready();
 
+let win;
+
 newWindowButton.onclick = function() {
   var url = document.getElementById('url').value;
   var windowName = document.getElementById('name').value;
   var isChild = document.getElementById('child').checked;
   appReady.then(() => {
-    // eslint-disable-next-line no-new
-    new ssf.Window(url, windowName, {
+    win = new ssf.Window(url, windowName, {
       child: isChild
     });
   });
 };
+
+var closeWindow = document.getElementById('close-window');
+
+closeWindow.onclick = () => {
+  win.close();
```

Comment:
if appReady never resolved, this will be undefined. I know this shouldn't happen, but perhaps it would be clearer if all of the code within this file sat within the appReady promise `then` block?